packages/sdk-react/src/components/receivables/InvoiceDetails/ExistingInvoiceDetails/useExistingInvoiceDetails.tsx

```diff
@@ -133,16 +161,26 @@ export function useExistingInvoiceDetails({
     );
   }, [deliveryMethod, receivable?.status]);
 
+  const isEditButtonVisible = useMemo(() => {
+    return (
```

Comment:
Probably, we have to check the user's permissions too.

Reply:
We don't handle permissions for receivables (at all) right now, unfortunately :(

Reply:
Do you mean there is no support on the backend for differentiating permissions?

Reply:
On BE side it's supported I guess, but on FE side there is no implementation of it

Reply:
Not, sure, but probably [this is it](https://github.com/team-monite/monite-sdk/blob/bde068d520a604abf069d910e48711a36fe58039/packages/sdk-react/src/components/receivables/InvoiceDetails/ExistingInvoiceDetails/ExistingReceivableDetails.tsx#L77-L81).

Reply:
`ExistingReceivableDetails` works only for `Quotes` & `Credit Notes` it has outdated UI and have been made more than a year ago and shouldn't be supported
https://github.com/team-monite/monite-sdk/blob/1cba52101f1691de8f7027b27da07e260b6b9147/packages/sdk-react/src/components/receivables/InvoiceDetails/ExistingInvoiceDetails/ExistingReceivableDetails.tsx#L60

All improvements are made only for `Invoices`. Currently `Invoices` doesn't support backend permissions

I don't want to add to this task which is called `Provide an ability to edit invoice` functionality to check permissions because it's out of scope of this task.

I created a task to support BE permissions for a new component ([task link](https://monite.atlassian.net/browse/DEV-10818)). Will it work for you in that case? To make it in a separate task

Reply:
Sure, btw, which actions are not supported?

Reply:
What do you mean? Permissions? All. All permissions are not supported on FE side for a new component